docs/modules/data-structures/partials/nav.adoc

```diff
@@ -28,19 +28,19 @@
 *** xref:data-structures:queue.adoc[]
 *** xref:data-structures:priority-queue.adoc[]
 *** xref:data-structures:multimap.adoc[]
+*** xref:data-structures:replicated-map.adoc[]
 *** xref:data-structures:set.adoc[]
 *** xref:data-structures:list.adoc[]
 *** xref:data-structures:ringbuffer.adoc[]
+*** xref:data-structures:flake-id-generator.adoc[]
+*** xref:data-structures:pn-counter.adoc[]
 ** CP Data Structures
 *** xref:data-structures:cpmap.adoc[]
 *** xref:data-structures:fencedlock.adoc[]
 *** xref:data-structures:iatomiclong.adoc[]
-*** xref:data-structures:isemaphore.adoc[]
 *** xref:data-structures:iatomicreference.adoc[]
 *** xref:data-structures:icountdownlatch.adoc[]
-*** xref:data-structures:pn-counter.adoc[]
-*** xref:data-structures:flake-id-generator.adoc[]
-*** xref:data-structures:replicated-map.adoc[]
+*** xref:data-structures:isemaphore.adoc[]
 ** Streaming Data Structures
 *** xref:data-structures:event-journal.adoc[]
 ** AI/ML Data Structures
```

Comment:
I can't suggest changes as you've not changed this section but you should change line 49 to 

**** xref:data-structures:vector-search-overview.adoc[Data Structure Design] 

This is the fix I merged today that K mentioned.

Or let the merge handle it, but one to watch out for!